extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -48,7 +48,6 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverBabRecoveryCountries';
 		$vars[] = 'wgAdDriverBeachfrontBidderCountries';
 		$vars[] = 'wgAdDriverBottomLeaderBoardMegaCountries';
-		$vars[] = 'wgAdDriverBottomLeaderBoardOnMobileCountries';
```

Comment:
We shouldn't remove this instant global in the same PR. IG cache will be invalidated earlier and we'll disable BLB on mobile for 24h. Am I right?

Reply:
Hmm, that is true. I'm gonna split this into two PR.